src/styles/blocks/button.scss

```
@@ -0,0 +1,49 @@
+.button {
+  display: flex;
+  height: 56px;
+  justify-content: center;
+  align-items: center;
+  gap: 4px;
+  width: 100%;
+  border: 2px solid $main-color;
+  border-radius: 32px;
+
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 100%;
+  letter-spacing: 0.02em;
+  text-decoration: none;
+  color: $main-color;
+  text-transform: uppercase;
+  transition: color 0.3s, background-color 0.3s, display 0.3s;
+
+  &:hover {
+    color: white;
+    background-color: #f9512e;
+  }
+
+  &:hover &__cookie {
+    filter: brightness(0) invert(1);
+  }
+
```

try to rewrite use nesting
```suggestion
  &__cookie {
    &:hover {
      filter: brightness(0) invert(1);
    }
  }

```

---

Якщо я перепишу як ти пропонуєш, то іконка буде міняти колір тільки тоді коли я безпосередньо наведу на неї, я написала таким чином, щоб іконка мінялась коли я наводжу на кнопку. Тому вважаю за доцільне залишити цей кусочок як є. Як вважаєш?

---

Agree 👍